### PR TITLE
fix: add check for chat attachments

### PR DESCRIPTION
**Pull request by scmmishra**
There might be a case where the chat is initialised as an empty object without the `attachments` array present. This PR handles that

Ref: https://github.com/chatwoot/chatwoot/pull/7156/

### CODE REVIEW

**Review by iamsivin — APPROVED** [review]
Tested locally and LGTM

File: app/javascript/dashboard/store/modules/conversations/index.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ export const mutations = {
       selectedChatId: conversationId,
     });
 
-    if (!chat) return;
+    if (!chat?.attachments) return;
```
Comment (pranavrajs) [inline app/javascript/dashboard/store/modules/conversations/index.js:154]: Can you identify the case where the chat is initialized without attachments?

Comment (marcomarinho) [reply app/javascript/dashboard/store/modules/conversations/index.js:154]: I do have this same issue where chat is initialized without attachments, mainly with inline images.. will test this and give feedback aswell

Comment (pranavrajs) [reply app/javascript/dashboard/store/modules/conversations/index.js:154]: Thanks! Appreciate it.

Comment (marcomarinho) [reply app/javascript/dashboard/store/modules/conversations/index.js:154]: Doesn't work in my use case. Inline images still not attached.

**Comment by scmmishra** [comment]
Fixed here: https://github.com/chatwoot/chatwoot/pull/9784

**Pull request closed without merging**